src/com/google/enterprise/adaptor/DocIdSender.java

```diff
@@ -355,8 +458,8 @@ public GroupPrincipal pushGroupDefinitions(
     for (int ntries = 1; keepGoing; ntries++) {
       try {
         log.info("sending groups to GSA host name: " + config.getGsaHostname());
-        fileSender.sendGroups(feedSourceName,
-            groupsDefXml, config.isServerToUseCompression());
+        fileSender.sendGroups(feedSourceName, feedType.toString(), groupsDefXml,
```

Comment:
It's me, the git history pedant! Why modify the second line if you don't have to?